static/js/lib/validation/profile.js

```diff
@@ -78,6 +80,15 @@ export const personalValidation = (profile: Profile) => {
   if (!moment(profile.date_of_birth).isBefore(moment(), 'day')) {
     errors.date_of_birth = personalMessages.date_of_birth;
   }
+  if (shouldRenderRomanizedFields(profile)) {
+    if (!profile.romanized_first_name || !CP1252_REGEX.test(profile.romanized_first_name)) {
+      errors.romanized_first_name = "CP-1252 string is required";
```

Comment:
I think we can use the same error message in both cases (blank and invalid characters)